### DISCUSSION TITLE
Text alignment and word wrap

### DISCUSSION THREAD

**Discussion by sytone** [RafaelGB/obsidian-db-folder#342] [Ideas]
I looked around but could not see a clear answer/plan on this. Now any text is centered in the column. Are there plans to allow the alignment of the text to be changed along with the ability to wrap text, so you do not have to scroll to see it?

Can this be done now with CSS as an option? Awesome plugin and being able to change alignment, wrap and longer term the padding in the cells would be nice. I guess wrapping the text is the primary issue I hit now.

**Comment by RafaelGB** [comment]
Word wrap is enabled when you use the normal and wide cell size on table settings. Disabled for compact mode.

About the text alignment ,every column type has its own style. It's a design decision.

Maybe in the future, just for text type, we can add more custom options, sure! but its not a priority right now

**Reply by robrecord** [reply]
Please could you make left-alignment the default for all columns, for easier scanning. All tables I've encountered have left-aligned text, unless they are in the heading row (centred) or a number column (right). Centred text is very hard to read for normal data.
